src/FluentValidation/DefaultValidatorExtensions.cs

```diff
@@ -1085,5 +1085,32 @@ public static class DefaultValidatorExtensions {
 			action(innerValidator.RuleForEach(x => x));
 			return ruleBuilder.SetValidator(innerValidator);
 		}
+
+		/// <summary>
+		/// Defines a enum value validator on the current rule builder that ensures that the specific value is a valid enum value.
+		/// </summary>
+		/// <typeparam name="T">Type of Enum being validated</typeparam>
+		/// <typeparam name="TProperty">Type of property being validated</typeparam>
+		/// <param name="ruleBuilder">The rule builder on which the validator should be defined</param>
+		/// <param name="enumType">The enum whose the string should match any name</param>
+		/// <returns></returns>
+		public static IRuleBuilderOptions<T, string> IsEnumName<T>(this IRuleBuilder<T, string> ruleBuilder, Type enumType)
+		{
+			return ruleBuilder.SetValidator(new StringEnumValidator(enumType));
+		}
+
+		/// <summary>
+		/// Defines a enum value validator on the current rule builder that ensures that the specific value is a valid enum value.
+		/// </summary>
+		/// <typeparam name="T">Type of Enum being validated</typeparam>
+		/// <typeparam name="TProperty">Type of property being validated</typeparam>
+		/// <param name="ruleBuilder">The rule builder on which the validator should be defined</param>
+		/// <param name="enumType">The enum whose the string should match any name</param>
+		/// <param name="caseSensitive">If the comparison between the string and the enum names should be case sensitive</param>
+		/// <returns></returns>
+		public static IRuleBuilderOptions<T, string> IsEnumName<T>(this IRuleBuilder<T, string> ruleBuilder, Type enumType, bool caseSensitive)

```

Comment:
We don't need 2 overloads for this. If `caseSensitive` is made an optional parameter with a default value then we only need a single overload.

Reply:
I like to separate methods into overloads with more parameters to simplify the first signature that appears within the IntelliSense, but all right for the optional parameter.

The most common way to send data over web apis nowadays is JSON, and JSON data is often camelCase. That's why I thought case insensitive by default would be less surprising. But with the optional parameter the default value will be clear, so let's go for case sensitive.